packages/core/src/hosted-form/hosted-form-order-data.ts

```diff
@@ -17,7 +18,9 @@ export default interface HostedFormOrderData {
     config?: Config;
     order?: Order;
     orderMeta?: OrderMeta;
-    payment?: (HostedCreditCardInstrument | HostedVaultedInstrument) & PaymentInstrumentMeta;
+    payment?:
+        | ((HostedCreditCardInstrument | HostedVaultedInstrument) & PaymentInstrumentMeta)
+        | NonceInstrument;
```

Comment:
This one appears to be payment method specific as well; could you kindly consider the Square approach?
https://github.com/bigcommerce/checkout-sdk-js/blob/b445659272efda3f381f1ed7332d8adef5da566d/packages/core/src/payment/strategies/square/square-form.ts#L154

Reply:
You can import `NonceInstrument` from `@bigcommerce/checkout-sdk/payment-integration-api`, but you need to add it to the export first [here](https://github.com/bigcommerce/checkout-sdk-js/blob/b445659272efda3f381f1ed7332d8adef5da566d/packages/payment-integration-api/src/index.ts#L61).